### ISSUE TITLE
OTG over tcpip mode 

### ISSUE THREAD

**Issue by oyekanwahab** [Genymobile/scrcpy#3453]
- [ ] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Is your feature request related to a problem? Please describe.**
I have been using scrcpy for  a very long time, all my phones are always rooted, so I always use scrcpy with tcpip, I just discovered that there is an otg mode, which is only accessible with USB

**Describe the solution you'd like**
Is it possible to enable this mode on tcpip mode, since the ADB daemon is not being used by the USB on tcpip  mode? 

**Describe alternatives you've considered**

**Additional context**

**Comment by rom1v** [comment]
No because we can only send HID events on Android using the AOA protocol (over USB).

**Comment by oyekanwahab** [comment]
ok.
 I was thinking that it was using ADB 
 
 I was thinking you wanted to implement what Deskdock was doing using ADB for mouse and keyboard input

**Comment by twaik** [comment]
Actually it is possible with root + uinput.

**Comment by oyekanwahab** [comment]
I root already on my device, can you show me how

**Comment by oyekanwahab** [comment]
Are you able to get it to work yourself

**Comment by twaik** [comment]
@rom1v it seems like uinput works fine over ADB TCPIP connection. I can work with https://github.com/Blub/netevent tool cross-compiled to aarch64 with all devices except Xiaomi. I think it can be used to implement capturing PC's keyboard and mouse.

**Comment by rom1v** [comment]
Since this issue, UHID mode has been implemented: https://github.com/Genymobile/scrcpy/blob/master/doc/keyboard.md#uhid
And the feature to control the device without video playback has been added recently (on `dev` branch): https://github.com/Genymobile/scrcpy/pull/4868

**Comment by twaik** [comment]
What about emulating mouse this way?

**Comment by rom1v** [comment]
It's also implemented.

**Comment by twaik** [comment]
You are right, keyboard works after installing last version from release page. But for some reason mouse does not work. I mean cursor is intercepted, but I do not see it on screen. Also clicks make some weird action I can not really identify. Phone reacts like I swipe up.

**Comment by rom1v** [comment]
> but I do not see it on screen

On some devices, the cursor is not displayed on lock screen.

Does it work on the home screen? (run `scrcpy -M`)

**Comment by twaik** [comment]
It does not.

**Comment by twaik** [comment]
Should I record a video?
